### PR TITLE
Collapse long container state reason messages

### PR DESCRIPTION
**Pull request by spadgett**
See https://github.com/openshift/origin/issues/2572

kubernetes_describer_example

### CODE REVIEW

**Comment by spadgett** [comment]
@jwforres ptal

File: views/container-state.html [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
+    <div ng-switch-when="terminated">
       Terminated
       <span ng-if="stateDescription.finishedAt">at {{stateDescription.finishedAt | date:'medium'}}</span>
       <span ng-if="stateDescription.exitCode">with exit code {{stateDescription.exitCode}}</span>
-      <span ng-if="stateDescription.reason">({{stateDescription.reason}})</span>
-    </span>
-    <span ng-switch-default>{{state}}</span>
-  </span>
-</span>
+      <div ng-if="stateDescription.reason" style="overflow: hidden; text-overflow: ellipsis;">
```
Comment (jwforres) [inline views/container-state.html:21]: does this result in a double ellipsis? the css ellipsis and the one added by the collapse-long-text ?

seems like this gets us into a state where the text is cut off and there is no way to see the full text

**Comment by jwforres** [comment]
while we are making changes in the object describer, can we also fix https://github.com/openshift/origin/issues/4238

File: views/container-state.html (hunk 1 above)
Comment (spadgett) [inline views/container-state.html:21]: No, only one ellipsis. You can still click the text to expand. Only the long, unbreakable lines get an ellipsis (e.g., pull specs).

Comment (spadgett) [inline views/container-state.html:21]: For example,

kubernetes_describer_example

Comment (spadgett) [inline views/container-state.html:21]: Same as above, but collapsed. You can click anywhere on the text to expand.

kubernetes_describer_example

**Comment by spadgett** [comment]
Sure, I'll look at it.

**Pull request closed without merging**